### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the name of the search edit box  in "Settings  dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1473]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to and select File->Settings.
3. In "settings" dialog , tab to and select plugin tree item.
4. Tab to search edit box and observe the Jaws and NVDA behaviour

**Actual Result:**
Jaws and NVDA is not announcing the name of the search edit box

**Jaws behaviour:**
Edit
Type in text

**NVDA Behaviour:**
edit
blank

**Expected Result:**
Jaws and NVDA should announce the name of the edit field
Name should be search

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![settings-edit-no name](https://user-images.githubusercontent.com/22956130/36103571-b65feb74-1035-11e8-8b8f-0a42a71fb127.jpg)
![bbb9e057](https://user-images.githubusercontent.com/22956130/36103574-ba3c1e3e-1035-11e8-95bf-7bb39bb34ae4.jpg)
![settings-edit-explorer](https://user-images.githubusercontent.com/22956130/36103580-bcaaf816-1035-11e8-9bf7-32d05c1794a5.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by soumyatippapuram** [comment]
This issue is still reproducing in the latest test pass
(IntelliJ IDEA 2019.1.3  Build #IU-191.7479.19)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749919
